__tests__/buildx/bake.test.itg.ts

```diff
@@ -35,14 +36,31 @@ maybe('getDefinition', () => {
     [
       'https://github.com/docker/buildx.git#v0.10.4',
       ['binaries-cross'],
-      path.join(fixturesDir, 'bake-buildx-0.10.4-binaries-cross.json')
+      path.join(fixturesDir, 'bake-buildx-0.10.4-binaries-cross.json'),
+      '>=0.6.0',
     ],
-  ])('given %p', async (source: string, targets: string[], out: string) => {
-    const bake = new Bake();
-    const expectedDef = <BakeDefinition>JSON.parse(fs.readFileSync(out, {encoding: 'utf-8'}).trim())
-    expect(await bake.getDefinition({
-      source: source,
-      targets: targets
-    })).toEqual(expectedDef);
+    [
+      'https://github.com/docker/test-docker-action.git#remote-private',
+      ['default'],
+      path.join(fixturesDir, 'bake-buildx-0.10.4-binaries-cross.json'),
+      '>=0.14.0',
+    ]
```

Comment:
Needs follow-up to bump buildx version in our `dev.Dockerfile` and `test` workflow to next 0.14.0 when available so this test match the buildx version constraint and is not skipped.